### ISSUE TITLE
Needs conversion to Layout Management

### ISSUE THREAD

**Issue by scottmc** [HaikuArchives/Becasso#5]
There's numerous cases of text clipping and buttons cut off etc.  This should probably be done is steps are there are a larger number of files.

**Comment by owenca** [comment]
A good first step would be converting these two windows to using Layout:
![becasso](https://user-images.githubusercontent.com/6434360/34368960-1f61fb42-ea6d-11e7-9818-15b7c9088243.png)
![new canvas](https://user-images.githubusercontent.com/6434360/34368970-2f7f3940-ea6d-11e7-9ac3-0a710d94b2f5.png)
![tip of the day](https://user-images.githubusercontent.com/6434360/34368986-401823d4-ea6d-11e7-8e91-b89575785f12.png)

**Comment by scottmc** [comment]
Yes, that would be a good start.  We would rather get a lot of small pull requests, fixing one or two files at a time, instead of one larger one changing 20+

**Comment by janus2** [comment]
I have some changes to the layout if somebody want to start this task please let me know. I will commit what is stable and discard what is in progress.

**Comment by janus2** [comment]
PR #19 add Size, Resize, Preferences windows.
Preferences use custome slider needs that to scale well.
